java/ql/src/semmle/code/java/frameworks/javase/URI.qll

```diff
@@ -0,0 +1,43 @@
+import java
+import semmle.code.java.dataflow.FlowSources
+
+/** Any expresion or call which returns a new URI.*/
+abstract class UriCreation extends Top {
```

Comment:
Let's move all the contents of this file to `Networking.qll` - that already contains `TypeUri`.